src/C/gdalwarper.jl

```diff
@@ -16,15 +16,15 @@
                          void * pValidityMask,
                          int * pbOutAllValid) -> CPLErr
 """
-function GDALWarpNoDataMasker(pMaskFuncArg,nBandCount::Cint,eType::GDALDataType,nXOff::Cint,nYOff::Cint,nXSize::Cint,nYSize::Cint,papabyImageData,bMaskIsFloat::Cint,pValidityMask,pbOutAllValid)
-    ccall((:GDALWarpNoDataMasker,libgdal),CPLErr,(Ptr{Void},Cint,GDALDataType,Cint,Cint,Cint,Cint,Ptr{Ptr{GByte}},Cint,Ptr{Void},Ptr{Cint}),pMaskFuncArg,nBandCount,eType,nXOff,nYOff,nXSize,nYSize,papabyImageData,bMaskIsFloat,pValidityMask,pbOutAllValid)
+function GDALWarpNoDataMasker(pMaskFuncArg, nBandCount::Cint, eType::GDALDataType, nXOff::Cint, nYOff::Cint, nXSize::Cint, nYSize::Cint, papabyImageData, bMaskIsFloat::Cint, pValidityMask, pbOutAllValid)
+    ccall((:GDALWarpNoDataMasker, libgdal), CPLErr, (Ptr{Void}, Cint, GDALDataType, Cint, Cint, Cint, Cint, Ptr{Ptr{GByte}}, Cint, Ptr{Void}, Ptr{Cint}), pMaskFuncArg, nBandCount, eType, nXOff, nYOff, nXSize, nYSize, papabyImageData, bMaskIsFloat, pValidityMask, pbOutAllValid)
 end
 
 
 """
     GDALWarpDstAlphaMasker(void * pMaskFuncArg,
                            int nBandCount,
-                           GDALDataType eType,
+                           GDALDataType,
```

Comment:
seems to have dropped some argument names?

Reply:
I see this information is no longer in the doxygen generated XML that I use to retrieve this information:
```xml
        <param>
          <type>int</type>
          <declname>nBandCount</declname>
        </param>
        <param>
          <type><ref refid="gdal_8h_1a22e22ce0a55036a96f652765793fb7a4" kindref="member">GDALDataType</ref></type>
        </param>
```

gdalwarper.cpp:
```cpp
GDALWarpDstAlphaMasker( void *pMaskFuncArg, int nBandCount,
                        CPL_UNUSED GDALDataType /* eType */,
```

gdalwarper.h
```cpp
GDALWarpDstAlphaMasker( void *pMaskFuncArg, int nBandCount, GDALDataType eType,
                        int nXOff, int nYOff, int nXSize, int nYSize,
```

I guess this is left out by doxygen because of the `/* eType */` in the cpp file. Either that is new, or the current doxygen version behaves differently.

Perhaps something to pick up in the future.

Reply:
yeah sure